README.md

```diff
@@ -1 +1,53 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+This is the first project in the APIs week at Microverse. It is a basic leaderboard with the use of custom Api from Microverse capstone Project
+
+![screenshot](./lead.png)
+
+## Built With
+
+- HTML5
+- CSS
+- Javascript
+- Webpack
+- npm
+
+## Live Demo
+
+[Live Demo Link](https://delhinrharl.github.io/Leaderboard/)
```

Comment:
- [ ] kindly deploy the project online and update the live deploy link in the readme.